README.md

```diff
@@ -0,0 +1,121 @@
+    * [Assign a Group to an Application](#assign-a-group-to-an-application)
+  * [System Log](#system-log)
+    * [Get Logs](#get-logs)
+  * [Call other API Endpoints](#call-other-api-endpoints)
+* [Collection](#collection)
+  * [each](#each)
+  * [subscribe](#subscribeconfig)
+* [Configuration](#configuration)
+* [Caching](#caching)
+* [Rate Limiting](#rate-limiting)
+  * [Built-In Retry](#built-in-retry)
+  * [Manual Retry](#manual-retry)
+* [Request Executor](#request-executor)
+  * [Default Request Executor](#default-request-executor)
+  * [Base Request Executor](#base-request-executor) -->
+
+## Contributing
+
+See [CONTRIBUTING.md](CONTRIBUTING.md) if you would like to propose changes to this library.
+
+[devforum]: https://devforum.okta.com/
+[Sessions: Create Session with Session Token]: https://developer.okta.com/docs/api/resources/sessions.html#create-session-with-session-token
+[Sessions: Session Properties]: https://developer.okta.com/docs/api/resources/sessions.html#session-properties
+[Sessions: Session Operations]: https://developer.okta.com/docs/api/resources/sessions.html#session-operations
+[Applications]: https://developer.okta.com/docs/api/resources/apps/
+[Applications: Application User Profile]: https://developer.okta.com/docs/api/resources/apps/#application-user-profile-object
+[Applications: Add Application]: https://developer.okta.com/docs/api/resources/apps/#add-application
+[Applications: User Operations]:https://developer.okta.com/docs/api/resources/apps/#application-user-operations
+[Basic Authentication Application]: https://developer.okta.com/docs/api/resources/apps/#add-basic-authentication-application
+[Client]: https://developer.okta.com/okta-sdk-nodejs/jsdocs/Client.html
+<!-- [DefaultRequestExecutor]: src/default-request-executor.js -->
+[Groups: Add Group]: https://developer.okta.com/docs/api/resources/groups.html#add-group
+<!-- [isomorphic-fetch]: https://github.com/matthew-andrews/isomorphic-fetch -->
+[Okta Developer Forum]: https://devforum.okta.com/
+[Okta Platform API]: https://developer.okta.com/docs/api/getting_started/api_test_client.html
+[Pagination]: https://developer.okta.com/docs/api/getting_started/design_principles.html#pagination
+[Rate Limiting at Okta]: https://developer.okta.com/docs/api/getting_started/rate-limits
+<!-- [RequestExecutor]: src/request-executor.js -->
+[System Log API]: https://developer.okta.com/docs/api/resources/system_log
+[Users API Reference]: https://developer.okta.com/docs/api/resources/users.html
+[Users: Create User]: https://developer.okta.com/docs/api/resources/users.html#create-user
+[Users: Get User]: https://developer.okta.com/docs/api/resources/users.html#get-user
+[Users: Lifecycle Operations]: https://developer.okta.com/docs/api/resources/users.html#lifecycle-operations
+[Users: List Users]: https://developer.okta.com/docs/api/resources/users.html#list-users
+[Users: Update User]: https://developer.okta.com/docs/api/resources/users.html#update-user
```

Comment:
We have a pretty well defined structure for Readme.md files at this point.  Again, refer to the go sdk repo, or .net repo for the structure.  I will dig up our wiki on it as well and point you to it.

Reply:
Updated